azure-pipelines.yml

```diff
@@ -181,3 +183,40 @@ jobs:
           SCREENER_API_KEY: $(screener.key)
 
       - template: .devops/templates/cleanup.yml
+
+  - job: ScreenerVNext
+    displayName: Screener @fluentui/react-components
+    workspace:
+      clean: all
+    steps:
+      - template: .devops/templates/tools.yml
+
+      - task: Bash@3
+        inputs:
+          filePath: yarn-ci.sh
+        displayName: yarn
+
+      - script: |
+          yarn workspace @fluentui/vr-tests-react-components screener:build
+        displayName: build vr-tests-react-components storybook
+
+      - task: AzureUpload@2
+        displayName: Upload @fluentui/react-components VR test site
+        inputs:
+          azureSubscription: $(azureSubscription)
+          BlobPrefix: $(deployBasePath)/react-components-screener
+          CacheControl: 'public, max-age=600000'
+          ContainerName: '$web'
+          SourcePath: 'apps/vr-tests-react-components/dist/storybook'
+          storage: $(azureStorage)
+
+      # Don't use lage here because it eats long output for reasons that are hard to debug
+      # - script: |
+      #     yarn workspace @fluentui/vr-tests-react-components screener
```

Comment:
Not running this yet, since there is a small code change needed so that the `just-scripts screener` task handles more than one baseUrl. I would like to do that in a follow up PR so the changes don't get lost in this bigger one